tests/assert-jobs/index/main.go

```diff
@@ -28,25 +26,26 @@ const (
 	flagAssertCountIndices = "assert-count-indices"
 	flagAssertCountDocs    = "assert-count-docs"
 	flagJaegerService      = "jaeger-service"
+	flagCertificatePath    = "certificate-path"
 	flagVerbose            = "verbose"
 )
 
 func filterIndices(indices *[]elasticsearch.EsIndex, pattern string) ([]elasticsearch.EsIndex, error) {
 	regexPattern, err := regexp.Compile(pattern)
 	if err != nil {
-		return nil, fmt.Errorf(fmt.Sprintf("There was a problem with the pattern: %s", err))
+		return nil, fmt.Errorf("There was a problem with the pattern: %s", err)
```

Comment:
Do we use capitalized in the rest of the repo? Otherwise i would recommend to stick to lowercase versions. 

Found this here: https://github.com/golang/go/wiki/CodeReviewComments#error-strings

What do you think?

Reply:
I didn't know about this. I'll fix it. Thanks!